src/cdk/table/table.ts

```diff
@@ -80,6 +81,19 @@ import {
 } from './table-errors';
 import {CDK_TABLE} from './tokens';
 
+
+/**
+ * Enables the recycle view repeater strategy, which reduces rendering latency. Not compatible with
+ * tables that animate rows.
+ */
+@Directive({
+  selector: 'cdk-table[cdkRecycleRows], table[cdk-table][cdkRecycleRows]',
```

Comment:
```suggestion
  selector: 'cdk-table[recycleRows], table[cdk-table][recycleRows]',
```
I would do just `recycleRows` since we're conceptually pretending that this is an option on `CdkTable`, the properties of which wouldn't be prefixed. Same for the `mat` version

Reply:
Done. Updated here and in the mat table.